x-pack/plugins/fleet/server/services/preconfiguration/outputs.ts

```diff
@@ -99,25 +107,79 @@ export async function createOrUpdatePreconfiguredOutputs(
       }
 
       const isUpdateWithNewData =
-        existingOutput && isPreconfiguredOutputDifferentFromCurrent(existingOutput, data);
-
-      if (isCreate) {
-        logger.debug(`Creating output ${output.id}`);
-        await outputService.create(soClient, esClient, data, { id, fromPreconfiguration: true });
-      } else if (isUpdateWithNewData) {
-        logger.debug(`Updating output ${output.id}`);
-        await outputService.update(soClient, esClient, id, data, { fromPreconfiguration: true });
-        // Bump revision of all policies using that output
-        if (outputData.is_default || outputData.is_default_monitoring) {
-          await agentPolicyService.bumpAllAgentPolicies(soClient, esClient);
-        } else {
-          await agentPolicyService.bumpAllAgentPoliciesForOutput(soClient, esClient, id);
+        existingOutput && (await isPreconfiguredOutputDifferentFromCurrent(existingOutput, data));
+
+      if (isCreate || isUpdateWithNewData) {
+        const secretHashes = await hashSecrets(output);
+
+        if (isCreate) {
+          logger.debug(`Creating output ${output.id}`);
```

Comment:
I know this is preexisting code, but could we add a note to these debug logs that these are preconfigured outputs we're updating, just to help with debugging and support cases? e.g. 

```ts
logger.debug(`Creating preconfigured output ${output.id}`)
```

Same for the logs below.